### ISSUE TITLE
Some questions about decode

### ISSUE THREAD

**Issue by Sunshine-spec-spec** [C0ldstudy/Argus#5]
Thank you so much for sharing your work.May I ask if you can solve some of my doubts ?

I noticed a new DEC module mentioned in section 4.4 of the paper, where can the implementation of this part of the function be found in the code?

I notice that there are **inner_forward_sm** and **self.out = nn.Sequential(nn.Linear(self.zdim, self.zdim), nn.Softmax(dim=1))** functions in the file argus.py. It seems that these two parts are not used in the decode process.

Maybe DEC is already included in the current feature, but I may not understand it.Thank you again for taking the time.
